### ISSUE TITLE
Future recruitment info when applications are closed

### ISSUE THREAD

**Issue by srijonsaha** [ubclaunchpad/ubclaunchpad.com#118]
When applications are closed and Join Us button is disabled, we should show information on when future recruitment is happening. It would be good have that information show up in the first section, alongside the club description so people can't miss it.

**Comment by AmirNaghibi** [comment]
Here is content idea! we can use the same format/wording.

This is a scholarship company that I'm currently working at and they only accept applications one a year.

![Screen Shot 2020-04-26 at 11 22 47 AM](https://user-images.githubusercontent.com/20001191/80316041-50559000-87b0-11ea-9c5e-214b3f0646d0.png)

**Comment by bobheadxi** [comment]
That looks like a big too big/too wordy for our needs - I think what @srijonsaha has in mind is replacing the "Fall 2020 applications are now open!" in the example with something like "We are not currently accepting new applications, but keep an eye on this page for updates!"

![](https://raw.githubusercontent.com/ubclaunchpad/ubclaunchpad.com/master/.static/homepage.png)

It would be a simple `v-else` element with the alternative text [here](https://sourcegraph.com/github.com/ubclaunchpad/ubclaunchpad.com/-/blob/src/sections/Feature.vue?subtree=true#L32)

**Comment by AmirNaghibi** [comment]
Agree! @bobheadxi maybe also mention the next recruitment cycle month. 

@ubclaunchpad/design 
@ubclaunchpad/strategy
